src/Resources/config/services.yml

```diff
@@ -12,6 +12,14 @@ services:
             - { name: kernel.event_listener, event: kernel.controller, method: onKernelController, priority: 254 }
             - { name: kernel.event_listener, event: kernel.finish_request, method: onKernelFinishRequest, priority: -254 }
 
+    contao.listener.handle_exception:
+        class: Contao\CoreBundle\EventListener\ExceptionListener
+        tags:
+            - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: 200 }
```

Comment:
I must be before `twig.exception_listener` and I guess before `security.exception_listener` as well but I am unsure about the latter one, maybe @Toflar or @aschempp know more what checks might be needed before.